file-formats/intf/zif_aff_types_v1.intf.abap

```diff
@@ -13,20 +13,15 @@ INTERFACE zif_aff_types_v1 PUBLIC.
     END OF ENUM abap_language_version.
 
   TYPES:
-    "! Head to be included in each object type
-    BEGIN OF head,
-      "! Location of the corresponding JSON schema
-      schema TYPE string,
```

Comment:
Why do you delete the schema in the abap type? It's still part of the json schema.

Reply:
versioning deferred to https://github.com/SAP/abap-file-formats/issues/53

the ABAP type is the source of truth going forward

Reply:
I have just talked to @Christopher-Hermann. 
He is also convinced, now. I.e., we check whether we can get rid of the schema from the ABAP type. But as of now, we would keep it in the json-schema and the json.